chia/wallet/puzzles/dao_lockup.clvm

```diff
@@ -41,9 +41,11 @@
     (if conditions
       (if (= (f (f conditions)) CREATE_COIN)  ; this guarantees that the new coin is obeying the rules - other coins are banned to avoid re-voting
         (if (= (f (r (f conditions))) vote_added_puzhash)
+          ; COMMENT: do we care? We're enforcing the amount is correct so what do we care if they make 0 CAT or use some value from a non voted CAT to make this?
```

Comment:
The spend cases here are:
- You are voting with all your value
- You are voting with part of your value

In both cases we want to make sure that you are only creating locked up cats, and that all of your value is accounted for.